.nais/kafka/behandler-dialogmelding-bestilling.yaml

```diff
@@ -31,3 +31,6 @@ spec:
     - team: disykefravar
       application: dvh-sykefravar-airflow-kafka
       access: read
+    - team: aap
+      application: dokumentinnhenting
+      access: write
```

Comment:
Burde vi utvidet modellen vår med noe flag om feks hvilket system ting kom fra?
Hvis det begynner å komme AAP-meldinger inn her, har vi behov for å skille de fra våre?

Reply:
Ja, vi burde kanskje legge til en kilde i DialogmeldingToBehandlerBestillingDTO?